src/components/Home.js

```diff
@@ -1,17 +1,35 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import CategoryDropdown from './CategoryDropdown';
 import BillCard from './Bill';
 import Container from '@material-ui/core/Container';
 
+// import { getBillsForCategory } from 'helpers/selectors';
+
 const Home = (props) => {
-	const billCards = props.bills.map((bill) => {
+	const [childCategory, setChildCategory] = useState(0);
+
+	useEffect(() => console.log('Category ID: ', childCategory), [childCategory]);
+
+	const bills = props.bills.filter((bill) => {
+		return childCategory === 0 ? bill : bill.categories.includes(childCategory);
+	});
+
+	const billCards = bills.map((bill) => {
 		return <BillCard key={bill.id} bill={bill} />;
 	});
 
+	const handleChange = (category) => {
+		setChildCategory(category);
+	};
+
 	return (
 		<div>
 			<Container maxWidth='sm'>
-				<CategoryDropdown categories={props.categories} />
+				<CategoryDropdown
+					categories={props.categories}
+					passCategory={setChildCategory}
+					onChange={handleChange}
```

Comment:
onChange props is not used in CategoryDropdown component. Can remove it.
The passCategory prop is already used to call  setChildCategory in this Home component.